recipes/repository.rb

```diff
@@ -195,10 +200,12 @@ def warn_deprecated_yumrepo_gpgkey
     action :install
     only_if { node['packages']['gnupg2'].nil? }
   end
-
   # Import new RPM key
   rpm_gpg_keys.each do |rpm_gpg_key|
     next unless node['datadog']["yumrepo_gpgkey_new_#{rpm_gpg_key[rpm_gpg_keys_short_fingerprint]}"]
+    if (agent_minor_version.nil? || agent_minor_version > 35) && rpm_gpg_key[rpm_gpg_keys_short_fingerprint] == 'e09422b3'
+      next
```

Comment:
do we add the same fingerprint for all keys? 🤔

Reply:
`rpm_gpg_keys_short_fingerprint` is an index 
`rpm_gpg_key` is a list of [elements](https://github.com/DataDog/chef-datadog/blob/b54194fc318463d2134d5a2031dc17a1cb74b85d/recipes/repository.rb#L53)  containing the name, the short fingerprint and the full one.